### PR TITLE
[ArticulatedSystemPlugin] New example

### PR DESCRIPTION
**Pull request by EulalieCoevoet**
Adds an example to the plugin ArticulatedSystemPlugin. 

image

______________________________________________________

By submitting this pull request, I acknowledge that  
**I have read, understand, and agree [SOFA Developer Certificate of Origin (DCO)](https://github.com/sofa-framework/sofa/blob/master/CONTRIBUTING.md#sofa-developer-certificate-of-origin-dco)**.
______________________________________________________

**Reviewers will merge this pull-request only if**  
- it builds with SUCCESS for all platforms on the CI.
- it does not generate new warnings.
- it does not generate new unit test failures.
- it does not generate new scene test failures.
- it does not break API compatibility.
- it is more than 1 week old (or has fast-merge label).

### CODE REVIEW

**Comment by fredroy** [comment]
I see that this PR is adding ~10MB of data (from the stl files).
I think we should wait for this plugin to be externalized (on an other repo) to merge this example, otherwise the history of this repo will contain those files, and the SOFA repo history is already so cluttered 🤪
what do you think @hugtalbot  ?

**Comment by EulalieCoevoet** [comment]
I've just cleaned the repo, removed unused mesh files.
It's 1.7MB of stl files now. Is it still too much ? I could also simplify the meshes but, yeah... it won't be as "pretty".

**Comment by fredroy** [comment]
[ci-build][with-all-tests]

File: applications/plugins/ArticulatedSystemPlugin/examples/ArticulatedArm/header.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
+
+def addHeader(rootNode):
+
+    rootNode.addObject('VisualStyle', displayFlags='showVisualModels hideBehaviorModels hideForceFields hideWireframe')
+    rootNode.addObject('RequiredPlugin', name='SofaPlugins', pluginName=['ArticulatedSystemPlugin',
+     'SofaGeneralAnimationLoop', 'SofaPython3', 'SofaSparseSolver', 'SofaPreconditioner', 'SuctionCup',
```
Comment (alxbilger) [inline applications/plugins/ArticulatedSystemPlugin/examples/ArticulatedArm/header.py]: `SuctionCup` cannot be loaded

**Review by alxbilger — CHANGES_REQUESTED** [review]

File: applications/plugins/ArticulatedSystemPlugin/examples/ArticulatedArm/header.py (hunk 1 above)
Comment (EulalieCoevoet) [reply applications/plugins/ArticulatedSystemPlugin/examples/ArticulatedArm/header.py]: Oops, forgot to remove that.

**Comment by guparan** [comment]
Scene test errors "extension (py) not handled" should be fixed now.

**Comment by EulalieCoevoet** [comment]
Okay I don't know why things appear like that. I think I just merged master into this branch. If someone knows please tell me. 

Anyway, I've cleaned the RequiredPlugin and added some forgotten changes.

**Comment by alxbilger** [comment]
Still some problems with the scene on the CI, probably because tkinter is not installed.

```
Python exception:   
  ModuleNotFoundError: No module named 'tkinter'
```

tkinter must be removed from the scene OR this scene should be ignored by the CI.

**Comment by EulalieCoevoet** [comment]
The GUI (which requires tkinter) is optional but really nice I think to interact with the arm. I've just disabled it by default, so if you go for making the CI ignoring a file, it should only be the script `robotGUI.py`.

**Comment by alxbilger** [comment]
@guparan @hugtalbot the scene has not been ignored despite the file .scene-tests

**Comment by fredroy** [comment]
I think it is because it is missing the relative path iteself, i.e it should be "ArticulatedArm/robotGUI.py"
I will modify it directly if I can

**Comment by damienmarchal** [comment]
Ready, so merge.

**Pull request merged**